sql-native/src/main/java/org/ballerinalang/sql/parameterprocessor/AbstractStatementParameterProcessor.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.sql.parameterprocessor;
+
+import io.ballerina.runtime.api.types.Field;
+import io.ballerina.runtime.api.values.BObject;
+import org.ballerinalang.sql.exception.ApplicationError;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * This class has abstract implementation of methods required convert ballerina types into SQL types and
+ * other methods that process the parameters of the statement.
```

Comment:
```suggestion
 * other methods that process the parameters of the statement.
 *
 * @since 0.5.6
 */
```

Reply:
I changed. Commit id is 8d544e07f231d8a5813e46316829f10717ddbf4a.